resonate/stores/local.py

```diff
@@ -161,6 +171,9 @@ def loop(self) -> None:
                         assert task.expiry is not None
                         timeout = task.expiry if timeout is None else min(task.expiry, timeout)
 
+                for schedule in self.schedules.scan():
+                    timeout = schedule.next_run_time * 1000 if timeout is None else min(timeout, schedule.next_run_time * 1000)
```

Comment:
```suggestion
                    timeout = schedule.next_run_time if timeout is None else min(timeout, schedule.next_run_time)
```

The store stores time as ints in milliseconds